### DISCUSSION TITLE
[Select] How to work the aria-select

### DISCUSSION THREAD

**Discussion by JeanLiima** [radix-ui/primitives#1737] [Help] [answered]
Hello everyone, I was making tests in my select, and I had some problems verifying if the options were selected.      
![image](https://user-images.githubusercontent.com/59691460/197181685-a8fe4b2e-6678-4bde-8bed-ae42fe5f7076.png)   
As you can see, it just changed the data-state, but the aria-select continuous false, are there something I was doing wrong ?

**Comment by benoitgrelard** [comment, marked as answer]
`aria-selected` will be true only for the selected item when focused. This is to deal with some issues with VoiceOver.

```tsx
// `isFocused` caveat fixes stuttering in VoiceOver
aria-selected={isSelected && isFocused}
```

What are you trying to test exactly? It sounds like your tests are way too low-level.

**Reply by JeanLiima** [reply]
I'm just doing some small tests to understand better how your select works and how much I can integrate it in project to change the old select I had. But makes sense what you said. Tks a lot!
Btw I'm just doing unit test of select, but we have another levels in the project, like integrated tests and e2e tests.
